packages/components/src/Banner/index.ts

```diff
@@ -1 +1,2 @@
-export { default, BannerIcon } from "./Banner";
+export { default } from "./Banner";
+export { BannerColor } from "./Banner";
```

Comment:
Should this be

```suggestion
export type { BannerColor } from "./Banner";
```

to help babel figure out it can be omitted from the JS?

Reply:
Oh good call! Looks like while this PR was out, that was fixed and then moved + renamed, so there's nothing to fix here, but thanks for taking a look and pointing that out 👍